bigbluebutton-html5/imports/ui/components/settings-loader/component.tsx

```
@@ -70,8 +73,13 @@ const SettingsLoader: React.FC<SettingsLoaderProps> = (props) => {
           .then((resp) => resp.json())
           .then((data: Response) => {
             clearTimeout(timeoutRef.current);
-            const settings = data?.meeting[0].clientSettings.clientSettingsJson;
+            const {
+              clientSettings,
+              ...staticData
+            } = data?.meeting[0];
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)